src/main/java/com/microsoft/sqlserver/jdbc/SQLServerClob.java

```diff
@@ -120,12 +124,19 @@ private String getDisplayClassName() {
      * @param logger
      */
     SQLServerClobBase(SQLServerConnection connection,
-            String data,
+            Object data,
```

Comment:
All calls to this Constructor has null as typeInfo, why is it even passed if it always going to be null? Is there any other place where an actual values is passed?